Logo/README.md

```diff
@@ -0,0 +1,29 @@
+# ClassicPress logos
+
+The main files are SVG ones.
+
+# Automatic conversion
+
+In the autogenerate folder you'll find automatically converted assets.
+
+The script checks that:
+
+- the paths are closed;
+- SVG title is file name (without extension).
```

Comment:
There's one other feature I'd like to add: the conversion script should automatically indent our master SVG files. This will make it much easier to review any changes.

I think this would make the "paths closed" and "title" tests mostly unnecessary since we'll be able to easily review the changes, but I guess we could leave them in too. What do you think?

Reply:
Nice! I've added this using `xmllint`.
Then if someone complains about file size (that is increased very little) it's easy to comment it out.